src/latis-python-client/latis.py

```diff
@@ -120,6 +228,20 @@ def __init__(self, latisInstance, name,
         self.buildQuery()
 
     def select(self, target="time", start="", end="", inclusive=True):
+        """
+        Define selection parameters for the dataset query.
+
+        Args:
+            target (str, optional): Target parameter for selection.
+                Defaults to 'time'.
+            start (str, optional): Start bound for selection. Empty by default.
+            end (str, optional): End bound for selection. Empty by default.
+            inclusive (bool, optional): If True, bounds are inclusive.
```

Comment:
> In the `select` method, the `inclusive` argument description can be a bit misleading as it reads as though both bounds would be inclusive if it's True. However, only the start bound will be inclusive in this case. It would be best to clarify that.

Reply:
Now the docstring is correct, but I don't think this is the right implementation. As is, the user can't perform a select where the start is inclusive but the end is exclusive, which is the most common use case (and should arguably be the default behavior). The right move might be to have two `inclusive` vars for start and end respectively, although I'm not sure.

Thoughts, @dlindhol @lindholc ?

Reply:
I think that the start should always be inclusive. It's the end that is typically the problem. I would default to exclusive (to avoid the double hit when stringing together multiple contiguous requests).

Reply:
See also #18.

Reply:
There are indeed cases where one might want the start to be exclusive. I suppose we shouldn't limit LaTiS capabilities.

Reply:
Since issue #18 exists what should be implemented in this PR? I can leave it how it is for now or maybe just add a second bool to independently toggle the start and end bounds. @sapols